modules/get-started/pages/whats-new.adoc

```diff
@@ -1,79 +1,95 @@
+* Changes to clusters, users, topics, ACLs, or consumer groups
+* Any time a user is authenticated
+* Any time a topic is written to or read from (requires explicit opt-in)
+* HTTP requests for the Schema Registry, HTTP Proxy, and Admin APIs
 
-- xref:manage:tiered-storage.adoc#view-space-usage[View space usage]: You can now use `rpk cluster logdirs describe` to get details about Tiered Storage space usage in both object storage and local disk.
+Auditing events can be stored in a topic, ensuring their retention for a specified period, even from Redpanda administrators. Auditing is compatible with the Open Cybersecurity Schema Framework (OCSF) and works with industry-standard tools, such as Splunk, Sumo Logic, and AWS.
 
-== Controller snapshots
+== Whole cluster restore
 
-Redpanda now loads the controller log from a snapshot on startup. xref:./architecture.adoc#controller-partition-and-snapshots[Controller snapshots] save the current cluster metadata state to disk, which significantly improves startup times of brokers in long-running Redpanda clusters. For example, with a partition that has moved several times, a snapshot can restore the latest state without replaying every move command.
+With Tiered Storage enabled, you can do a whole cluster restore to recover data (including its metadata) from a failed cluster to a new cluster. This is a simpler and cheaper alternative to active-active replication, for example with MirrorMaker 2. Use this recovery method to restore your application to the latest functional state as quickly as possible.
 
-== Delete records from a topic
+== Recovery mode 
 
-Redpanda enables you to xref:develop:config-topics.adoc#delete-records-from-a-topic[delete data from the beginning of a partition up to a specific offset]. The offset represents the true creation time of the event, not the time when it was stored by Redpanda. Deleting records frees up space in local disk and in object storage, which is especially helpful if your producers are pushing more data than you anticipated when sizing your storage infrastructure, or if you want to implement a data retention policy aligned with a particular business event and not based on age or size. There are different ways to delete records from a topic, including using the `rpk topic trim-prefix` command or using the DeleteRecords Kafka API with Kafka clients.
+xref:manage:recovery-mode.adoc[Recovery mode] lets you repair and restore a failed cluster that cannot start normally due to issues such as a system crash or out-of-memory errors. In recovery mode, Redpanda limits functionality to cluster configuration changes and other manual administrative actions so that you can repair the cluster.
```

Comment:
'lets you repair and revive a failed cluster'

Let's avoid using 'restore' as it collides with whole cluster restore which has a specific meaning of loading archived content into a new cluster.